src/lib/constants.ts

```diff
@@ -0,0 +1,37 @@
+export type METHOD = "GET" | "PUT" | "POST" | "DELETE" | "PATCH" | "OPTIONS";
```

Comment:
Non-blocking: `OPTIONS` is not a verb you'd add here. It has to be an allowed method for CORS to even function.